src/services/CertificateGenerationService.ts

```diff
@@ -70,11 +70,19 @@ class CertificateGenerationService {
     const config: IMOTConfig = this.config.getMOTConfig();
     const iConfig: IInvokeConfig = this.config.getInvokeConfig();
     const testType: any = testResult.testTypes;
+    const { STOP_WELSH_GEN } = process.env;
 
-    // Find out if Welsh certificate is needed
-    const testStations = await this.getTestStations();
-    const testStationPostcode = this.getThisTestStation(testStations, testResult.testStationPNumber);
-    const isTestStationWelsh = testStationPostcode ? await this.lookupPostcode(testStationPostcode) : false;
+    let isTestStationWelsh = false;
+
+    // Circumvent bilingual certificate generation and logic if EV set to TRUE
+    if (STOP_WELSH_GEN === undefined || STOP_WELSH_GEN.toUpperCase() === "FALSE") {
+      // Find out if Welsh certificate is needed
+      const testStations = await this.getTestStations();
+      const testStationPostcode = this.getThisTestStation(testStations, testResult.testStationPNumber);
+      isTestStationWelsh = testStationPostcode ? await this.lookupPostcode(testStationPostcode) : false;
+    } else {
+      console.log(`Welsh certificate generation deactivated via environment variable set to ${STOP_WELSH_GEN}`);
+    }
```

Comment:
might want to switch this logic to make it easier to read

```suggestion
   let isTestStationWelsh = true;

if (STOP_WELSH_GEN && STOP_WELSH_GEN.toUpperCase() === "TRUE") {
  console.log(`Welsh certificate generation deactivated via environment variable set to ${STOP_WELSH_GEN}`);
} else {
  const testStations = await this.getTestStations();
  const testStationPostcode = this.getThisTestStation(testStations, testResult.testStationPNumber);
  isTestStationWelsh = testStationPostcode ? await this.lookupPostcode(testStationPostcode) : false;
}
```

Reply:
Happy to flip it if you think it reads better, will make the change in the PR from VTAWELSH-1 -> develop, as will be a number of changes to make following the rebase anyway.